PWGGA/GammaConvBase/AliCaloPhotonCuts.cxx

```diff
@@ -677,6 +681,10 @@ void AliCaloPhotonCuts::InitCutHistograms(TString name){
       fHistClusterEtavsPhiAfterAcc->GetXaxis()->SetTitle("#varphi (rad)");
       fHistClusterEtavsPhiAfterAcc->GetYaxis()->SetTitle("#eta");
       fHistograms->Add(fHistClusterEtavsPhiAfterAcc);
+      fHistClusterEtavsPhiBackground  = new TH2F(Form("EtaPhi_Background %s",GetCutNumber().Data()),"EtaPhi_Background",nEmcalPhiBins,EmcalPhiBins,nEmcalEtaBins,EmcalEtaBins);
```

Comment:
What about creating these histograms only if ")->DoGammaSwappForBg()" is running?

Reply:
I added a flag and a setter in the photon cuts and modified the AddTasks for Calo and ConvCalo accordingly. Now the histogram is only initialized if one uses the cluster swapping background. Is this solution okay?